### PR TITLE
Benchmark framework implementation and add YuNet for face detection, DB for text detection, CRNN for text recognition

### PR DESCRIPTION
**Pull request by fengyuentau**
- [x] Benchmark framework implementation.
- [x] Add implementation and benchmark for YuNet (face detection) .
- [x] Add implementation and benchmark for DB (text detection).
- [x] Add implementation and benchmark for CRNN (text recognition).

### CODE REVIEW

**Comment by fengyuentau** [comment]
cc @kaingwade @zihaomu

**Review by zihaomu — COMMENTED** [review]
Thanks for your contribution!

**Comment by fengyuentau** [comment]
Copy right needs to be changed to `Copyright (C) 2021, Shenzhen Institute of Artificial Intelligence and Robotics for Society, all rights reserved.`.

File: models/face_detection_yunet/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# YuNet
+
+YuNet is a light-weight, fast and accurate face detection model, which achieves 0.834(easy), 0.824(medium), 0.708(hard) on the WIDER Face validation set.
```
Comment (kaingwade) [inline models/face_detection_yunet/README.md:3]: @fengyuentau easy -> AP_easy

File: README.md [hunk 2]
```diff
@@ -2,6 +2,53 @@
...
+<!-- 
+- [Text Detection](#text-detection)
+- [Text Recognition](#text-recognition)
+-->
+
+Platforms:
+- `CPU x86_64`: INTEL CPU i7-5930K @ 3.50GHz, 6 cores, 12 threads.
+- `CPU ARM`: Raspberry 4B, BCM2711B0 @ 1.5GHz (Cortex A-72), 4 cores, 4 threads.
+<!--
+- `GPU CUDA`: Jetson Nano.
```
Comment (kaingwade) [inline README.md:20]: @fengyuentau can add GPU CUDA: NVIDIA® Jetson Nano™ B01, 128-core Maxwell, Quad-core ARM A57 @ 1.43 GHz

File: README.md [hunk 3]
```diff
@@ -2,6 +2,53 @@
...
+-->
+
+### Face Detection
+
+- Input size: width = 160, height = 120.
+- Accuracy: Results on the [WIDER Face dataset](http://shuoyang1213.me/WIDERFACE/) are collected from the source of each model.
+
+| Model | Accuracy | CPU x86_64 (ms) | CPU ARM (ms) |
+|-------|----------|-----------------|--------------|
+| [YuNet](./models/face_detection_yunet)  | E: 83.4, M: 82.4, H: 70.8 | 2.1749 | TBD |
```
Comment (kaingwade) [inline README.md:30]: @fengyuentau keep 2 decimal places?

**Comment by fengyuentau** [comment]
cc @vpisarev

File: README.md [hunk 4]
```diff
@@ -2,6 +2,51 @@
...
+
+
+### Face Detection
+
+- Input size: width = 160, height = 120.
+- Accuracy: Results on the [WIDER Face dataset](http://shuoyang1213.me/WIDERFACE/) are collected from the source of each model.
+
+| Model | Accuracy | CPU x86_64 (ms) | CPU ARM (ms) | GPU CUDA (ms) |
+|-------|----------|-----------------|--------------|---------------|
+| [YuNet](./models/face_detection_yunet)  | E: 83.4, M: 82.4, H: 70.8 | 2.17 | TBD | TBD |
```
Comment (zihaomu) [inline README.md:27]: A more detailed description of the test method is needed. For example, whether to test multiple times to eliminate accidental effects.

File: benchmark/README.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+Benchmarking different models in the zoo.
+
+Data for benchmarking will be downloaded and loaded in [data](./data) based on given config.
+
+## Preparation
+
+1. Install `python >= 3.6`.
+2. Install dependencies: `pip install -r requirements.txt`.
+
+## Benchmarking
```
Comment (kaingwade) [inline benchmark/README.md:12]: can description of how the time is measured.

File: README.md [hunk 6]
```diff
@@ -2,6 +2,55 @@
...
+
+
+### Text Recognition
+
+- Input size: width = 100, height = 32
+- Accuracy: Results on the [ICDAR15](https://iapr.org/archives/icdar2015/index.html%3Fp=254.html) are collected from the source of each model.
+
+| Model | Accuracy | CPU x86_64 (ms) | CPU ARM (ms) | GPU CUDA (ms) |
+|-------|----------|-----------------|--------------|---------------|
+| [CRNN](./models/text_recognition_crnn) | 66.28% | 23.23 | 235.87 | TBD |
```
Comment (zihaomu) [inline README.md:51]: Can we combine these three tables into one? Others look good.

File: benchmark/README.md (hunk 5 above)
Comment (fengyuentau) [reply benchmark/README.md:14]: Done.

**Pull request merged**